.github/workflows/neard_release.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       - name: Checkout nearcore repository
         if: ${{ github.event_name != 'workflow_dispatch'}}
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Documented in the [readme](https://github.com/actions/checkout#usage)

Reply:
now I know :)